### PR TITLE
custom-tokens

### PR DESCRIPTION
**Pull request by enriquesouza**

### CODE REVIEW

File: VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift [hunk 1]
```diff
@@ -43,6 +43,10 @@ extension ChainDetailView {
                 if viewModel.hasTokens(chain: group.chain) {
                     addButton
                 }
+
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift:46]: you need to add this to MacOS as well

File: VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift [hunk 2]
```diff
@@ -60,6 +64,14 @@ extension ChainDetailView {
         }
     }
     
+    var addCustomTokenButton: some View {
+        Button {
+            sheetType = .customToken
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift:69]: Also , can we also hardcode some of the most common tokens on Solana into our token store?

Comment (enriquesouza) [reply VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift:69]: Yes, def we can.

File: VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift (hunk 1 above)
Comment (enriquesouza) [reply VultisigApp/VultisigApp/iOS/View/ChainDetailView+iOS.swift:46]: Tks

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**